go-controller/pkg/ovn/namespace.go

```diff
@@ -49,20 +54,26 @@ func (oc *Controller) waitForNamespaceEvent(namespace string) error {
 	return nil
 }
 
-func (oc *Controller) addPodToNamespaceAddressSet(ns string, ip net.IP) {
+func (oc *Controller) addPodToNamespaceAddressSet(ns string, ip net.IP,
+	logicalPort string) {
+
 	mutex := oc.getNamespaceLock(ns)
 	if mutex == nil {
 		return
 	}
+
+	if oc.namespacePolicies[ns] == nil {
```

Comment:
@dceara I think this has to be after the defer, because getNamespaceLock() actually locks the returned mutex before returning.

Reply:
@dcbw Ah, sorry for that, i messed up the rebase. I'll fix it soon.